cylc/flow/etc/examples/datetime-cycling/flow.cylc

```diff
@@ -0,0 +1,44 @@
+[meta]
+    title = Datetime Cycling
+    description = """
+        A basic cycling workflow which runs the same set of tasks over
+        and over. Each cycle will be given a datetime identifier.
+
+        The task "a" will wait until the real-world (or wallclock) time passes
+        the cycle time.
+
+        Try changing the "initial cycle point" to "previous(00T00) - P1D" to
+        see how this works.
+    """
+
+[scheduling]
+    # start the workflow cycling at 00:00 this morning
```

Comment:
```suggestion
    # Start the workflow cycling at 00:00 this morning.
    # It will catch up to the current time.
```

Reply:
adequately explained IMO.

Reply:
Actually IMO we could do with taking pains not to imply any suggestion of wall-clock triggering in a comment explaining the initial cycle point. Granted this _is_ a wall-clock triggering workflow, but that is decoupled from the initial cycle point
```suggestion
    # set the start of the graph to 00:00 this morning
```

Reply:
I'm just trying to get inside the head of a new user who is developing the common misconception that datetime cycling means wall-clock triggering, and I think they could read it that way. I think "set the start of the graph to" more clearly decouples the two than "start the workflow cycling at"

Reply:
I tend to agree with @MetRonnie. I've run into more than a few new users who struggle to understand that datetime cycling is not automatically tied to the wall clock, so it wouldn't hurt to avoid saying anything that could easily be misconstrued that way.

Reply:
"start the workflow cycling" and "set the start of the graph" are equivalent though.

Since this IS a clock-triggered example, it would be more correct to say, "start the workflow at ..."?!

Reply:
The workflow and the graph are 2 separate things though. Strictly speaking the workflow starts the moment you play it...

Reply:
No, but I don't see it as any better either. I was just pushing back on something I see as too nitpicky to warrant comment. Ability to care depleted.